.npmrc

```diff
@@ -1 +1,2 @@
-@micromedio:registry=https://npm.pkg.github.com
+@micromedio:registry=https://npm.pkg.github.com/
+//npm.pkg.github.com/:_authToken=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
```

Comment:
não né

Reply:
Não, rs.

Reply:
Esse pr foi inútil pq no final das contas fizemos outro.